src/config/i18n.ts

```diff
@@ -1,28 +1,32 @@
-import i18n from 'i18next';
+/* eslint-disable @typescript-eslint/explicit-function-return-type */
 
-import { initReactI18next } from 'react-i18next';
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
+import { initReactI18next, useTranslation } from 'react-i18next';
 
+import buildI18n, { namespaces } from '@graasp/translations';
+
+import ar from '../langs/ar.json';
+import de from '../langs/de.json';
 import en from '../langs/en.json';
 import fr from '../langs/fr.json';
+import it from '../langs/it.json';
+
+const i18n = buildI18n().use(initReactI18next);
```

Comment:
This function allows to define the default namespace to use. Set it to be the Text analysis namespace.

Reply:
Okay I will redo